### DISCUSSION TITLE
Remote Service for "chargingProfile"

### DISCUSSION THREAD

**Discussion by weymann** [bimmerconnected/bimmer_connected#391] [Feature request]
In the MyBMW App it is possible to adapt the `chargingProfile` with different settings, times and days. While the actual profile is delivered in the `status `section it's not possible to send different settings to BMW API.

Do you have any hint how to update  `chargingProfile`  for the car?
[vehicles-de.zip](https://github.com/bimmerconnected/bimmer_connected/files/7784346/vehicles-de.zip)

**Comment by rikroe** [comment]
It should be possible using a not yet implemented remote service command, see https://github.com/bimmerconnected/bimmer_connected/discussions/387.

The main issue would probably be creating a model of the data that the BMW API understands (this could be partly MITM'ed if one's car supports this, which mine doesn't).

I had the idea of adding a CLI command that supports arbitrary execution e.g. using `command_id` and `command_data` parameters. 
Of course there wouldn't be any validation whatsoever, so it could mix things up. I'll let that sink in if that could make sense.

**Comment by RutgerBeyen** [comment]
+1, I'm also interested in this feature. Having the option to switch bewteen DelayedCharging and ImmediateCharging would already be great.
